contracts/crowdsale/ZxcCrowdsale.sol

```diff
@@ -266,29 +301,22 @@ contract ZxcCrowdsale is
   /**
    * @dev Calculate amount of tokens for a given wei amount. Apply special bonuses depending on
    * @param weiAmount Amount of wei for token purchase.
-   * the time of the purchase. If purchase is outside of every token sale window then revert.
-   * @return number of tokens per wei amount with possible token bonus
+   * @param bonusPercent Percentage of bonus tokens.
+   * @return Number of tokens with possible bonus.
    */
-  function getTokenAmount(uint256 weiAmount)
+  function getTokenAmount(uint256 weiAmount, uint256 bonusPercent)
```

Comment:
Every argument should be in its own line.

Reply:
e82ca41